content/en/continuous_integration/guides/identify_highest_impact_jobs_with_critical_path.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Identify CI Jobs on the Critical Path to reduce the Pipeline Duration
+description: Learn how to identify those CI Jobs that are on the critical path to improve the duration of your CI Pipelines.
+further_reading:
+  - link: "/continuous_integration/search/#pipeline-details-and-executions"
+    tag: "Documentation"
+    text: "Learn how to search and manage your pipeline executions"
+  - link: "continuous_integration/search/#highlight-critical-path"
+    tag: "Documentation"
+    text: "Highlight critical path in your Pipeline Execution"
+---
+
+## Overview
+
+This guide explains how to identify the CI jobs that are on the critical path, with the goal of determining which CI jobs to focus on to reduce the overall duration of the CI pipelines.
+
+### Understanding the Critical Path in a CI Pipeline
+
+The critical path in a CI Pipeline is the longest sequence of CI Jobs that determines the total duration of that pipeline. In other words, it is the path through the DAG of CI Jobs that requires the most time to complete. You can only reduce the total duration of a CI Pipeline by reducing the duration of CI Jobs that are on the critical path.
+
+{{< img src="continuous_integration/critical_path_highlight_pipeline.png" alt="Highlight of jobs on the critical path in a pipeline execution." width="90%">}}
+
+However, CI jobs are typically executed in parallel with other jobs, meaning that the actual reduction in pipeline time is determined by the **exclusive time** a CI job spends on the CI runner during its execution.
+
+The exclusive time that a job is on the critical path represents the amount of time the CI runner has spent executing a specific job exclusively, excluding the execution time of other jobs that were running in parallel.
+
+{{< img src="continuous_integration/critical_path_highlight_pipeline_exclusive_time.png" alt="Highlight exclusive time of the jobs on the critical path in a pipeline execution." width="90%">}}
+
+If CI job A is on the critical path with a duration of 100ms and runs in parallel with CI job B, which takes 80ms, the exclusive time for job A on the critical path would be 20ms. This means the actual improvement in pipeline duration reduction would be 20ms.
+
+### Compatibility
+
+<div class="alert alert-info">Are you interested in critical path and your CI provider is not supported yet? Fill out <a href="https://forms.gle/zDgiDSGQxA1HgjXh6" target="_blank">this form</a>.</div>
+
+Filtering and Exclusive Time computation of CI Jobs on the Critical Path is available for the following CI Providers:
+* [GitLab][3]
+
+You can still [highlight which CI Jobs are on the critical path][4] using the Pipeline Execution detail view across all CI providers.
```

Comment:
Can you please add a link to this guide the other way around, in https://docs.datadoghq.com/continuous_integration/search/#highlight-critical-path ?

Reply:
Added